conda_forge_tick/update_sources.py

```diff
@@ -843,26 +846,6 @@ def get_version(self, url: str, node_attrs: AttrsTypedDict) -> Optional[str]:
         return latest
 
 
-# TODO: this one does not work because the atom feeds from libraries.io
-# all return 403 and I cannot find the correct URL to use
-# also url_contains is not defined
-# also package_name is not defined
-class LibrariesIO(VersionFromFeed):
```

Comment:
I think it's generally good practice not to keep dead code around. It's still in version control.